ChiantiPy/core/Continuum.py

```diff
@@ -440,13 +440,15 @@ def calculate_free_bound_emission(self, wavelength, include_abundance=True, incl
 
         # combine factors
         fb_emiss = prefactor*energy_over_temp_factor*sum_factor
+        #
         # include abundance, ionization equilibrium, photon conversion, emission measure
         if include_abundance:
             fb_emiss *= self.abundance
         if include_ioneq:
             fb_emiss *= self.ioneq_one(**kwargs)[:,np.newaxis]
         if self.emission_measure is not None:
-            fb_emiss *= self.emission_measure
+            for i, em in enumerate(self.emission_measure):
```

Comment:
When dealing with Numpy arrays, I think for loops should always be a last resort. This can be fixed using broadcasting in Numpy. So the loop should be replaced with

```Python
fb_emiss *= self.emission_measure[:,np.newaxis]
```